src/js/electron/main.js

```diff
@@ -55,21 +54,26 @@ async function main() {
     }
   }
 
-  app.on("ready", () => {
+  function onReady() {
     installExtensions()
```

Comment:
Not exactly related, but I noticed we call this but the function is all commented out. Any reason why we don't use those tools in dev? (the Redux devtools are particularly useful)

Reply:
Oh right. I think the redux dev tools was getting really slow. I'm open to trying them out again though.